README.md

```diff
@@ -55,6 +55,7 @@ module "localhost_function" {
 | bucket\_force\_destroy | When deleting the GCS bucket containing the cloud function, delete all objects in the bucket first. | bool | `"false"` | no |
 | bucket\_labels | A set of key/value label pairs to assign to the function source archive bucket. | map(string) | `<map>` | no |
 | bucket\_name | The name to apply to the bucket. Will default to a string of the function name. | string | `""` | no |
+| create\_bucket | Whether to create a new bucket or use an existing bucket with the bucket_name passed to the module. | bool | `"true"` | no |
```

Comment:
we should clarify that if `create_bucket=false` then name is expected in `bucket_name`

Reply:
Is this not clear? I referenced the bucket_name var in the description.

How about: "Whether to create a new bucket or use an existing one. If false, optional variable bucket_name is expected."

Reply:
Alternatively:

Whether to create a new bucket or use an existing one. If false, var.bucket_name should reference the name of the bucket to use.

Reply:
Latest commit contains by best shot at clear verbiage.

Reply:
ah I think that underscore didn't catch my eye earlier, even better now 👍